client/app/components/packages/pas-form/show.hbs

```diff
@@ -288,7 +318,7 @@
         </span>
       </p>
 
-      {{#if (eq @package.pasForm.dcpLanduseactiontype2 717170000)}}
+      {{#if @package.pasForm.dcpPleaseexplaintypeiienvreview}}
```

Comment:
fab! reads much better imo

Reply:
It's different logic. Instead of checking that the parent question's answer makes the input available in the edit view (which is potentially irrelevant here), we just check if the data we wanna show exists.

Reply:
okay. if it's a boolean and it's false it won't show. btw.